chibuezeayogu/React-Math-Skills-Game
# #160836841 Add game component

**What does the PR do?**
- Adds `Game Component`

**Description of Task to be completed?**
- write component test
- add game component
- create and bind component actions

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Math-Skills-Game.gitt)
-  cd into `React-Math-Skills-Game` and checkout to this branch `ft-add-game-component-160836841`
- run this command `npm start` to start the app.
- run this command `npm test` to ensure all test are passing.

**What are the relevant pivotal tracker stories?**
- [#160836841](https://www.pivotaltracker.com/story/show/160836841)

**Screenshots**
- N/A
